srcpkgs/python3-python-socks/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'python3-python-socks'
+pkgname=python3-python-socks
```

Comment:
I prefer to name it `python3-socks`. However, I prefer to include this package when it's actually need it.